script.module.tzlocal/lib/tzlocal/test_data/symlink_localtime/etc/localtime

```diff
@@ -0,0 +1 @@
+../usr/share/zoneinfo/Africa/Harare
```

Comment:
This includes a binary files I am not sure that are allowed:

script.module.tzlocal/lib/tzlocal/test_data/Harare
script.module.tzlocal/lib/tzlocal/test_data/localtime/etc/localtime